Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.17
+WORKDIR /
+COPY . .
+RUN make build-cli
+CMD ./bin/combo run
```

Comment:
I think we'd want to have a multi-stage Dockerfile here: build the CLI in one stage, and in the second copy that binary over so the overall image size is lower. This can likely be done in a follow-up though.

Reply:
Going to go ahead and break that into a separate PR.